src/store/models/network.ts

```diff
@@ -147,11 +156,14 @@ export interface NetworkModel {
     RootModel,
     Promise<Network>
   >;
+  autoMiners: { [id: number]: AutoMiner };
+  autoMine: Action<NetworkModel, { id: number; mode: AutoMineMode }>;
```

Comment:
This state would be good to move into the new autoMiner store model that you create.

Reply:
Can you please elaborate a little bit (considering the latest changed I've made essentially having moved all logic into the store)?